catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -393,6 +404,18 @@ private Double degreesToMeters(double distance) {
                 Distance.LinearUnit.KILOMETER).getAs(Distance.LinearUnit.METER);
     }
 
+    private Integer getSolrQueryRowsProperty() {
+        try {
+            Integer rows = Integer.valueOf(System.getProperty(SOLR_QUERY_ROWS_PROPERTY));
```

Comment:
I think @pklinef and myself were talking about leaving the property undocumented anyway. The chances of needing to tweak it for practical use cases is slim, but it's a good precaution to have for integration use cases that do heavy querying that's not necessarily targeting the browser. 

On that note I may double the value of `SOLR_QUERY_DEFAULT_ROWS` to be sure reasons for tweaking it remain slim. I like the new `getProperty` syntax so we don't have to mess with the `system.properties` file.